mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/core/zookeeper/CuratorService.java

```diff
@@ -90,10 +83,11 @@ public void stateChanged(CuratorFramework client, ConnectionState newState) {
     @Override
     public void start() {
         try {
-            isConnectedGauge.set(0L);
-            setupCuratorListener();
-            curator.start();
-            masterMonitor.startAsync().awaitRunning();
+            if(curator.getState() != CuratorFrameworkState.STARTED) {
```

Comment:
Starting the curator more than once throws an error. The same will happen if it's in `STOPPED`. I chose to check for not started because if you are stopped and want to be in the running state, that is not possible and I felt an exception should be thrown for that.